app/views/activities/_activity_feed.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="row t-activity-feed">
+  <div class="col-md-12 border-bottom">
+    <h2>Recent activity</h2>
+
+    <div class="activity-feed">
+      <div class="well js-message-form">
+        <%= form_for MessageActivity.new(appointment: appointment), url: appointment_activities_path(appointment), remote: true do |f| %>
+          <div class="input-group">
+            <%= f.text_field :message, placeholder: 'Add an action to keep others in the loop', class: 'form-control js-message t-message' %>
+            <span class="input-group-btn">
+              <%= f.button class: 'btn btn-primary btn-block t-submit-message', data: { disable_with: 'Adding message...' } do %>
+                <span class="glyphicon glyphicon-plus" aria-hidden="true"></span>
+                Add message
+              <% end %>
+            </span>
+          </div>
+        <% end %>
+      </div>
+      <ul class="activity-feed__list"
```

Comment:
At the moment this ends up rendering a line when there are no messages in the activity feed.

Would be great if we could have a `hide` class on the UL and then remove it when there are messages to show.